data/human/human missions.txt

```diff
@@ -5428,3 +5428,76 @@ mission "Deal Change 2"
+			choice
+				`	"Oh, so you guys did set up that regular shipment after all!"`
+					goto context
+
+				`	"Sure, how much mail are we talking about?"`
+					goto shipment
+
+				`	"Sorry, but I'm not heading in that direction."`
+					goto refuse
+
+			label context
+			`	The mail employee appears confused for a moment, leaning his head to the side slightly.
+			choice
+				`	"I once delivered a misposted package like that, and the clerk mentioned that it happened so often somebody should set up a regular delivery."`
+				`	"Never mind. What's the size of the shipment?"`
+					goto shipment
+			`	The post employee smiles and responds, "Oh! That was me, actually, back when I worked as a clerk for the post office. I was sick of always having that same conversation with so many pilots a day, so I went out and found a captain to set up a regular delivery with and made the proposal to my boss, who was just as sick of my and the Quicksilver postmaster's endless reports. Got a bonus and promotion for it; I'm now responsible for the delivery's management, instead! Anyways, are you interested in covering for them?"`
+			choice
+				`	"Yeah, what's the size of the shipment?"`
+					goto shipment
+				`	"Sorry, but I'm not able to work that into my plans."`
+					goto refuse
+
+			label shipment
+			`	The man takes a look at his tablet and says, "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`
+			`	"38 tons of misposted packages? Just this month?!" you ask.`
+			`	He nods. "Told you it happened often. And few of these packages even weigh over a kilo! Big shipments hardly get misposted, it's mostly private citizens who make such a mistake. Mostly. You'd be amazed!"`
+			`	You two shake hands and arrange for the misposted packages to be loaded onto the <ship>. Time to head to Quicksilver.`
+				accept
+
+			label refuse
+			`	The mail employee thanks you for your time, and continues his search for a willing captain. At least pilots that get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future.`
+				decline
+	on visit
+		dialog `You have reached <planet>, but your escort, loaded with the shipment, has not arrived. Better depart and wait for your escorts to arrive in this star system.`
+
+
+mission "Quicksilver Mixup [3]"
+	name "Misposted mail delivery"
+	deadline 10 1
+	landing
+	description "Return the misposted mail back to <destination>."
+	source "Quicksilver"
+	destination "Silver"
+	cargo "misposted mail" 29
+	to offer
+		has "Quicksilver Mixup [2]"
+	on offer
+		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to Silver. You, again, have 10 days to make the delivery.`
```

Comment:
```suggestion
		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to Silver. Again, you have 10 days to make the delivery.`
```
This is easier to read.

Reply:
I am guilty of loving inversions, heheheh
But yeah, i'll commit the simpler version. It is easier to read.